tests/unit_tests/test_elasticsearchstore.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import mock
+
+import mlflow_elasticsearchstore
+from mlflow_elasticsearchstore.elasticsearch_store import ElasticsearchStore
+from mlflow_elasticsearchstore.models import ElasticExperiment, ElasticRun, \
+    ElasticMetric, ElasticParam, ElasticTag
+
+
+@mock.patch('mlflow_elasticsearchstore.elasticsearch_store.ElasticsearchStore')
+@mock.patch('mlflow_elasticsearchstore.models.ElasticExperiment')
```

Comment:
You don't want to mock the classes you actually want to test, what you want to do is mock the elasticsearch_dsl library, you can assume it works (they probably have test in their repo).
So the principle is, you mock the elasticsearch library, and call the real functions from your code, and you check that the elasticsearch library functions have been called with the right parameters, and that your function returns the correct thing.